.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - '5.4'
   - '5.5'
   - '5.6'
+  - '7.0'
```

Comment:
I use PHP7, was expecting travis to be able to test it, but now i see it doesn't.